control_toolbox/include/control_toolbox/pid_ros.hpp

```diff
@@ -184,6 +220,38 @@ class PidROS
    */
   void set_gains(double p, double i, double d, double i_max, double i_min, bool antiwindup = false);
 
+  /*!
+   * \brief Initialize the PID controller and set the parameters
+   * \param p The proportional gain.
+   * \param i The integral gain.
+   * \param d The derivative gain.
+   * \param i_max The max integral windup.
+   * \param i_min The min integral windup.
+   * \param u_max Upper output clamp.
+   * \param u_min Lower output clamp.
+   * \param trk_tc Specifies the tracking time constant for the 'back_calculation'
+        and 'conditioning_technique' strategies. If set to 0.0 when one of these
+        strategies is selected, a recommended default value will be applied.
+   * \param saturation Enables output saturation. When true, the controller output is
+        clamped between u_max and u_min.
+   * \param antiwindup Anti-windup functionality. When set to true, limits
+        the integral error to prevent windup; otherwise, constrains the
+        integral contribution to the control output. i_max and
+        i_min are applied in both scenarios.
+   * \param antiwindup_strat Specifies the anti-windup strategy. Options: 'back_calculation',
+        'conditioning_technique', 'conditional_integration', or 'none'. Note that
+        the 'back_calculation' and 'conditioning_technique' strategies use the
+        tracking_time_constant parameter to tune the anti-windup behavior. When a strategy
+        other than 'none' is selected, it will override the controller's default
+        anti-windup behavior.
+   *
+   * \note New gains are not applied if i_min > i_max or if u_min_ > u_max_.
+   */
+  void set_gains(
+    double p, double i, double d, double i_max, double i_min, double u_max, double u_min,
+    double trk_tc, bool saturation = false, bool antiwindup = false,
+    std::string antiwindup_strat = "none");
```

Comment:
that's what I meant with default value
```suggestion
  void set_gains(
    double p, double i, double d, double i_max, double i_min, double u_max, double u_min,
    double trk_tc = 0.0, bool saturation = false, bool antiwindup = false,
    std::string antiwindup_strat = "none");
```